apps/api/v1/pages/api/audit-logs/_post.ts

```
@@ -0,0 +1,37 @@
+      throw new HttpError({
+        statusCode: 400,
+        message: `Missing required field: ${field}`,
+      });
+    }
+  }
+
+  try {
+    await AuditLogService.logEvent(body as AuditLogEvent);
+    return res.status(201).json({ ok: true });
+  } catch (error: unknown) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)